db/schema.rb

```diff
@@ -71,6 +71,14 @@
     t.index ["account_id", "target_account_id"], name: "index_blocks_on_account_id_and_target_account_id", unique: true, using: :btree
   end
 
+  create_table "boosts_mutes", force: :cascade do |t|
+    t.integer  "account_id",        null: false
+    t.integer  "target_account_id", null: false
+    t.datetime "created_at",        null: false
+    t.datetime "updated_at",        null: false
+    t.index ["account_id", "target_account_id"], name: "index_boosts_mutes_on_account_id_and_target_account_id", unique: true, using: :btree
+  end
+
```

Comment:
I'm sorry for lack of a word. You don't need to delete timestamps.

Delete the `boosts_mutes` table.